### DISCUSSION TITLE
Redis vs Express-sessions?

### DISCUSSION THREAD

**Discussion by kissingerbrian** [C-I-T-CLUB/CIT_CLUB_OFFICIAL_WEB_APP_-BACKEND#6] [Polls]
Which way would you want us to go? Redis or Express-sessions way?

**Comment by kissingerbrian** [comment]
I feel if we go redis we'll consider docker container because it's so easier to work with redis image in docker container than set it up locally

**Comment by Stephen-Kamau** [comment]
I think the same can also be done with express

**Reply by kissingerbrian** [reply]
I concur
